internal/controlplane/xds_cluster_test.go

```diff
@@ -542,6 +542,25 @@ func Test_buildCluster(t *testing.T) {
 	})
 }
 
+func Test_validateClusters(t *testing.T) {
+	type c []*envoy_config_cluster_v3.Cluster
+	testCases := []struct {
+		clusters    c
+		expectError bool
+	}{
+		{c{{Name: "one"}, {Name: "one"}}, true},
```

Comment:
Should we test the error case?

Reply:
this one is an error case (duplicate cluster names)

Reply:
Sorry, I mean the inverse. Basically to test both cases. Perhaps some test cases with different cases, and whitespace?